tools/ci/travis-install.py

```diff
@@ -53,7 +53,6 @@ def install(omp=False, mpi=False, phdf5=False, dagmc=False):
 
     # Build and install
     cmake_cmd.append('..')
-    print(' '.join(cmake_cmd))
```

Comment:
I would prefer to leave the `print` in there. This script is run by our automated CI (Travis) and the output of the `print` will show up in the test logs. It's helpful to see what args were passed to CMake in case of a build failure.

Reply:
Reverted - keeping it in for log detail clarity makes sense. I believe this one was flagged unlike others because it may have appeared more remnant of debugging.
******
<p align="left">
    <a href="https://www.pullrequest.com/">
        <img width="15%" src="https://static.pullrequest.com/images/dashboard/pullrequest-logo.svg">
    </a>
</p>